### ISSUE TITLE
Add support for UUID

### ISSUE THREAD

**Issue by hartungstenio** [simplejson/simplejson#321]
Should `simplejson` support encoding `UUID` objects to `str`?

If you think it should, I'm willing to implement it.

**Comment by etrepum** [comment]
I don't think that adding UUID to string is a good idea for a default. It's not really feasible to decode UUIDs from string, especially not by default, and there are several ways someone might want to format them.

**Comment by etrepum** [comment]
The problem isn’t necessarily uuid->string but the other way around, which requires something higher level like a schema. Even for uuid there are some choices for format, you might choose something more compact than dash delimited hex.
